churaverse-plugins-client/src/synchroBreakPlugin/ui/descriptionWindow/descriptionWindow.ts

```diff
@@ -26,7 +32,121 @@ export class DescriptionWindow implements IDescriptionWindow {
    * 説明ウィンドウの文章を更新する
    * @param text 更新する文章
    */
-  public setDescriptionText(text: string): void {
+  private setDescriptionText(text: string): void {
     this.element.innerHTML = text
   }
+
+  /**
+   * ゲーム開始時主催者サイドの文章更新処理
+   */
+  public setGameStartForHost(): void {
+    this.open()
+    this.setDescriptionText(
+      `${this.gameName}を開始しました。<br>あなたはゲームの管理者です。 <br>ターン数(1~10)を選択してください。`
+    )
```

Comment:
表示→Textを更新 としていますが、（おそらく同期的に処理されるので現状でもほぼ問題ないかと思いますが）一瞬更新前のメッセージが表示されたのちにText更新となりかねないので、Textを更新→表示 の順で処理した方が良い気がしますね
```suggestion
    this.setDescriptionText(
      `${this.gameName}を開始しました。<br>あなたはゲームの管理者です。 <br>ターン数(1~10)を選択してください。`
    )
    this.open()
```